From 16c10e076d9570bfe2446e985cc30e923fb39e6a Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Tue, 27 Jul 2021 17:17:24 -0500
Subject: [PATCH 1/1] ARM:dts:aspeed: Add Fan controllers to dts
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Adds Fan controllers to device tree. On Onyx, there are 4 controllers
controlling 5 fans each.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index 413f01b42314..b997469fed49 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -253,15 +253,21 @@
 		#address-cells = <1>;
 		#size-cells = <0>;

-		//  port 0 to 4 connected to 5 fans@0x4d
+		//  port 0 to 3 connected to 4 fan controllers which controls 5 fans each
 		i2c@0 {
-			#address-cells = <1>;
-			#size-cells = <0>;
 			reg = <0>;
-			emc2305@4d {
-				compatible = "smsc,emc2305";
-				reg = <0x4d>;
-			};
+		};
+
+		i2c@1 {
+			reg = <0>;
+		};
+
+		i2c@2 {
+			reg = <0>;
+		};
+
+		i2c@3 {
+			reg = <0>;
 		};

 		i2c@5 {
--
2.17.1

